clusterloader2/pkg/prometheus/manifests/pushgateway/deployment.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: monitoring
+  name: pushgateway
+  labels:
+    k8s-app: pushgateway
+  annotations:
+    prometheus.io/scrape: 'true'
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: pushgateway
+  template:
+    metadata:
+      name: pushgateway
+      labels:
+        k8s-app: pushgateway
+        visualize: "true"
+        run: pushgateway
```

Comment:
Do we need these two labels "visualize" and "run"?

Reply:
Not really, checked without those labels - it works.